.github/workflows/pr-merge.yml

```
@@ -0,0 +1,56 @@
+    branches: [main, conda]
+
+jobs:
+  notify-discord-merge:
+    name: Notify Discord (merged PR)
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Notify merge
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)